frontend/src/metabase/ui/components/overlays/Menu/Menu.styled.tsx

```diff
@@ -27,6 +27,10 @@ export const getMenuOverrides = (): MantineThemeOverride["components"] => ({
             color: theme.fn.themeColor("brand"),
           },
         },
+
```

Comment:
The override for the `Menu.Item:disabled -> color` that I mention in the PR description